### ISSUE TITLE
issues about the learning rate

### ISSUE THREAD

**Issue by h982639009** [xwjabc/hed#6]
Thanks for sharing your code and it works well.

I have some doubts about the learning rate. Why learning rate for conv5 is 100x of base learning rate, and classification layer(dsns) is 0.01x of base learning rate?  For fune-tuning,  learning rate for classification layer(which is trained from scratch) is usually set 10x compared to the backbone network parameters, right?

Besides, if I insert new layers into hed, how should I set the learning rate for them?

**Comment by xwjabc** [comment]
Hi. For the learning rate, I am sorry that I just simply re-use all the settings in the original HED repository and did not tune much about these hyper-parameters. You may change some of them and check effect after 1 or 2 epochs.
For the insertion of new layers, you can set 1x at first and check the results. Later modify it to 0.1x / 10x if results are bad.

**Comment by h982639009** [comment]
Many thanks to you! I'd have a try.
